### ISSUE TITLE
stats_mentor.go is required

### ISSUE THREAD

**Issue by snowden-fu** [kossiitkgp/KWoC-Backend#184]
like `stats_student.go`, `stats_mentor.go` is also required, matching frontend task [mentor_stats page](https://kossiitkgp.org/KWoC-Backend/)

**Comment by harshkhandeparkar** [comment]
Thank you for your suggestion @snowden-fu! Could you also list the statistics that could be returned by this endpoint?

**Comment by snowden-fu** [comment]
@harshkhandeparkar As you mentioned in [the frontend issue](https://github.com/kossiitkgp/KWoC-Frontend/issues/194), these data below should be returned. 
> each mentor's total PRs, total commits, total lines changed, and total number of projects.

**Comment by Devansh-bit** [comment]
I would like to work on this issue.

**Comment by harshkhandeparkar** [comment]
> @harshkhandeparkar As you mentioned in [the frontend issue](https://github.com/kossiitkgp/KWoC-Frontend/issues/194), these data below should be returned.
> 
> > each mentor's total PRs, total commits, total lines changed, and total number of projects.

Can you edit your issue body and add this? Also, do you want to work on this issue?

**Comment by snowden-fu** [comment]
> > @harshkhandeparkar As you mentioned in [the frontend issue](https://github.com/kossiitkgp/KWoC-Frontend/issues/194), these data below should be returned.
> > > each mentor's total PRs, total commits, total lines changed, and total number of projects.
> 
> Can you edit your issue body and add this? Also, do you want to work on this issue?

No, mate, I don't know anything about Go lang, but I want to do frontend part of student and mentor stats.

**Comment by Devansh-bit** [comment]
Hi, I would like to work on this issue. 
Also, a mentor's commits, lines added etc means the commits and lines added in all of their projects right?

**Comment by harshkhandeparkar** [comment]
> Hi, I would like to work on this issue. Also, a mentor's commits, lines added etc means the commits and lines added in all of their projects right?

Assigned and yes. But we do want to improve it next year using separate tables.
